Application/Activities/ActivityDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Application.Profiles;
+
+namespace Application.Activities
+{
+    public class ActivityDto
+    {
+        public Guid Id { get; set; }
+        public string Title { get; set; }
+        public DateTime Date { get; set; }
+        public string Description { get; set; }
+
+        public string Category { get; set; }
+
+        public string Location { get; set; }
+        public string HostUserName { get; set; }
+        public ICollection<Profile> Antendees { get; set; }
```

Comment:
What is a `Profile`? Can it mean multiple things, or only attendee? If it has a single meaning, let's keep the same name for the class and the property to avoid confusion.

Reply:
Typo on `Antendees`

Reply:
I renamed the Profile class into AtendeeProfile.